Makefile

```diff
@@ -1,14 +1,17 @@
 export GOOS=windows
 
-fmt:
```

Comment:
Just reordered this so that `build` will be the default action rather than `fmt`